src/config.rs

```diff
@@ -2207,16 +2207,13 @@ impl Default for BackupConfig {
     }
 }
 
-// TODO: `CdcConfig` is used by both `cdc` worker and `resolved ts` worker
-// should separate it into two configs
-#[derive(Clone, Serialize, Deserialize, PartialEq, Debug, Configuration)]
+#[derive(Clone, Serialize, Deserialize, PartialEq, Debug)]
 #[serde(default)]
 #[serde(rename_all = "kebab-case")]
 pub struct CdcConfig {
     pub min_ts_interval: ReadableDuration,
```

Comment:
Is it still a valid configuration given now `advance_ts_interval` is used?

Reply:
`advance_ts_interval` and `min_ts_interval` is used by different config